### PR TITLE
Add batch 7

### PR DESCRIPTION
**Pull request by Yullia**

### CODE REVIEW

File: snapshot.yaml [hunk 1]
```diff
@@ -1168,3 +1168,7 @@
   sha: f3693cffd03d7006ba8fc6159e7e9bc6dca5f13e
   tag: 3.7.0-1-gf3693cff
   url: https://github.com/CTFd/CTFd
+- id: SShaX0s/
+  sha: 40d49d9791cea8ad07904bdf901610ed5d8268ee
+  tag: None
+  url: https://github.com/microsoft/Git-Credential-Manager-for-Windows
```
Comment (babenek) [inline snapshot.yaml:1174]: MIT License

**Review by babenek — APPROVED** [review]
Approved

**Review by babenek — DISMISSED** [review]
Approved

**Review by babenek — APPROVED** [review]
Approved

**Review by xDizzix — DISMISSED** [review]

**Review by babenek — APPROVED** [review]
Approved

**Review by babenek — DISMISSED** [review]
Approved

**Review by iuriimet — DISMISSED** [review]

**Review by babenek — APPROVED** [review]
Approved after merge conflict fix

**Review by iuriimet — APPROVED** [review]

**Pull request merged**